docker-compose.yml

```diff
@@ -27,5 +28,6 @@ services:
       - POSTGRES_PASSWORD
       - POSTGRES_DB
       - POSTGRES_USER
+      - DB_HOST
     ports:
-      - '5432:5432'
\ No newline at end of file
+      - '5433:5432'
```

Comment:
why? how it helps current ticket?

Reply:
My bad was not working local cos i had local DB running on this port, but i think it wont work for everybody who is using both Docker and local